rest/resource-server/src/test/java/org/eclipse/sw360/rest/resourceserver/restdocs/ComponentSpecTest.java

```diff
@@ -764,6 +764,16 @@ public void should_document_get_component_attachment() throws Exception {
                 .andDo(this.documentationHandler.document());
     }
 
+    @Test
+    public void should_document_get_component_attachment_bundle() throws Exception {
+        String accessToken = TestHelper.getAccessToken(mockMvc, testUserId, testUserPassword);
+        mockMvc.perform(get("/api/components/" + angularComponent.getId() + "/attachments/download")
+                        .header("Authorization", "Bearer " + accessToken)
+                        .accept("application/*"))
```

Comment:
Please change this to "application/zip" as well

Reply:
Thanks, Changes done.